ci/ReleaseTool/ReleaseCommand.cs

```diff
@@ -263,6 +258,105 @@ private void FinalizeRelease(GitHubClient gitHubClient, GitClient gitClient, Rep
             CreatePRFromReleaseToSource(gitHubClient, gitHubRepo, repoUrl, gitRepoName, gitClient);
         }
 
+        private void CreatePRFromReleaseToSource(GitHubClient gitHubClient, Repository gitHubRepo, string repoUrl, string repoName, GitClient gitClient)
```

Comment:
Not changed but just moved up so the very long and difficult to read string literal file writing function comes last in the cs file.